packages/models-library/src/models_library/api_schemas_resource_usage_tracker/service_runs.py

```diff
@@ -17,6 +17,7 @@ class ServiceRunGet(BaseModel):
     wallet_id: WalletID | None
     wallet_name: str | None
     user_id: UserID
+    user_email: str
```

Comment:
It is highly recommended, specially for the external APIs that we use constraint strings (e.g. with limited characters or some pre-defined rule). This increases specially security. Moreover, it is not a big effort since we typically have alias for almost every important field. 

In tis case, it is not extremely dangerous since it is a read type but I would nonetheless annotate it as `EmailStr`. Same applies for some of other fields

Reply:
Will do in next iteration of improvements :+1: